lib/graphql/schema/member/has_arguments.rb

```diff
@@ -110,14 +114,23 @@ def coerce_arguments(parent_object, values, context)
                   arg_defn.prepare_value(parent_object, coerced_value, context: context)
                 end
 
-                kwarg_arguments[arg_defn.keyword] = prepared_value
+                kwarg_arguments[arg_key] = prepared_value
+                # TODO code smell to access such a deeply-nested constant in a distant module
+                argument_values[arg_key] = GraphQL::Execution::Interpreter::Arguments::ArgumentValue.new(
```

Comment:
I feel like this smell is telling me to figure out a way to use `Query::Arguments` here, despite my efforts to the contrary 😆

Reply:
I'm gonna leave it. I don't want to mess with `Query::Arguments`'s expectation that custom subclasses are built, or with it's normalize-to-string behavior.